backend/Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN mkdir jar
 # Copy jar file generated 
 COPY --from=maven /home/maven/target/moneytree-*.jar /home/maven/jar/moneytree.jar
 # Execute command once image started
-CMD ["java", "-jar", "/home/maven/jar/moneytree.jar"]
\ No newline at end of file
+CMD ["java", "-Dspring.profiles.active=prod", "-jar", "/home/maven/jar/moneytree.jar"]
```

Comment:
could use env variable instead of hardcoding, still works like that.

Reply:
I made it like that because we only have 2 profiles in the backend (dev and prod). If one day we expand on those, we can make the proper changes. That dockerfile is used to deploy so the profile will always be prod for now.